src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -22,8 +22,10 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.Abstractions" Version="1.0.0-preview" />
```

Comment:
I think there are some other subtle issues with bringing this package in here that we want to avoid if possible. Do you know exactly what classes we'd need to bring in manually? We may have a different approach we can take...

Reply:
Indeed. We don't want to have any WebJobs dependencies coming into those projects. 

Can you clarify what is required from this?

Reply:
FunctionMetadata and BindingMetadata + all the other types they use. The sdk uses this package and it simplified deserializing the json and simplified the code b/c of built in functions like .Create( raw JSON) for bindings. 

It's not necessary though - removed it in the next iteration and focused on just the information we needed for the grpc message and to pass the host-side validation checks.